source/features/rgh-linkify-features.tsx

```diff
@@ -5,13 +5,27 @@ import * as pageDetect from 'github-url-detection';
 import {wrap} from '../helpers/dom-utils';
 import features from '.';
 import {getNewFeatureName} from '../options-storage';
-import {isNotRefinedGitHubRepo} from '../github-helpers';
+import {isRefinedGitHubRepo} from '../github-helpers';
 import onConversationHeaderUpdate from '../github-events/on-conversation-header-update';
 
 function linkifyFeature(codeElement: HTMLElement): void {
 	const id = getNewFeatureName(codeElement.textContent!) as FeatureID;
-	if (features.list.includes(id) && !codeElement.closest('a')) {
-		wrap(codeElement, <a href={`/refined-github/refined-github/blob/main/source/features/${id}.tsx`}/>);
+	if (features.list.includes(id)) {
+		const href = `/refined-github/refined-github/blob/main/source/features/${id}.tsx`;
```

Comment:
Do we have a helper function for this? It's also generated in the options page.

Reply:
I added one

Reply:
Let's make one https://github.com/refined-github/refined-github/blob/6d5bda3b9db8b8d1514af40ff53628f198f4478d/source/options.tsx#L116